### ISSUE TITLE
The data model should be able to handle subtasking 

### ISSUE THREAD

**Issue by wkirschenmann** [aneoconsulting/ArmoniK#3]
The data model should include two new informations:
- the list of dependency tasks that need to be completed before launching the new task. 
- the parent task that has created the task

**Comment by wkirschenmann** [comment]
The parent task link will be implemented as a "subsession".
The new SessionId object will contain:
* _SessionId_, the id of the whole job session.
* _SubSessionId_, the id of the task that created the task.
